rllib/core/models/specs/checker.py

```diff
@@ -41,47 +41,37 @@ def convert_to_canonical_format(spec: SpecType) -> Union[Spec, SpecDict]:
         - a single constraint. The constraint can be a Spec object, a type, or None.
```

Comment:
This file is explicitly skipped in the `BUILD` file. Could you remove it from the list so that it's tested now?